test/controllers/update/update.cannot.use.controller.spec.ts

```diff
@@ -0,0 +1,31 @@
+jest.mock("ioredis");
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+
+import { NextFunction, Request, Response } from "express";
+import { expect, test, describe, jest } from "@jest/globals";
+import request from "supertest";
+
+import app from "../../../src/app";
+import * as config from "../../../src/config";
+
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { UPDATE_CANNOT_USE_SERVICE_HEADING } from "../../__mocks__/text.mock";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next());
+
+describe("UPDATE CANNOT USE controller", () => {
+  describe("GET tests", () => {
+    test(`renders the ${config.UPDATE_CANNOT_USE_PAGE} page`, async () => {
```

Comment:
also just type the URL you expect here instead of using `${config.UPDATE_CANNOT_USE_PAGE}`, or whatever you think. @conuirchgov discovered tests with variables in the names can't be run individually so trying to not use variables going forward